### DISCUSSION TITLE
Need help reading and converting MNG files

### DISCUSSION THREAD

**Discussion by jagroups** [ImageMagick/ImageMagick#4957] [Help]
I'm new to ImageMagick. I just wanted to be able to read and/or convert a MNG file.
Anyone that can help me, I'd appreciate it.

Thanks in advance.

**Comment by fmw42** [comment]
Convert MNG to what format?  All frames or just one frame?  What is your platform/OS?  Do you have ImageMagick installed and what version?

**Reply by jagroups** [reply]
Thanks for your reply.

I would like an avi file, all frames. I have Windows 7 Pro. I did install ImageMagick and tried following the instructions. Could not figure it out. I spent a few days at it. I don't know what version it was, I deleted it from my laptop.

Joe
